README.md

```diff
@@ -149,6 +149,22 @@ In this example, the `e3db.Client.query` method returns an iterator that contain
 
 See [the simple example code](https://github.com/tozny/e3db-python/blob/master/examples/simple.py) for runnable detailed examples.
 
+## Cipher Suite Selection
```

Comment:
Would appreciate documentation on how to build python w/ NIST ciphers; I understand it requires building python from source?

Could put in `DEVELOPMENT.md` if not appropriate here.

Reply:
Nothing special is needed to support the NIST ciphers. They'll all work out of the box. If you want to use FIPS-certified crypto under the hood, however, there is some extra work you have to do on the front-end. This will be documented in our internal docs repo.